template.rb

```diff
@@ -67,6 +65,10 @@ def apply_template!(template_root)
   post_default_addons_install
 
   # Add-ons - [Optional]
+  if yes?(install_addon_prompt('Rspec'))
+    delete_test_folder
+    apply '.template/addons/rspec/template.rb'
+  end
```

Comment:
In the case the addon is **not** selected, should we also remove the following?
- `./bin/test.sh` — as it depends on the `rspec` package
- `./docker-compose.test.yml` — as it is using the `test.sh` script